### ISSUE TITLE
cesiumWorkerBootstrapper does not have cache busting

### ISSUE THREAD

**Issue by jony89** [CesiumGS/cesium#9057]
Latest cesium version has changed the file `cesiumWorkerBootstrapper` 

yet under Build/Cesium/Workers/ the file `cesiumWorkerBootstrapper` does not have cache busting, i.e no hash to this file. most other files seems ok.

which caused to latest version to break until hard refresh is done.

**Comment by OmarShehata** [comment]
Doesn't this apply to all files in the library that are served statically and fetched at runtime? And that could be something the application can handle based on how you're serving those files/by serving them under a folder like `Cesium/1.71/Build/` etc.

**Comment by jony89** [comment]
> Doesn't this apply to all files in the library that are served statically and fetched at runtime?

True. yet, cesium is a bit different. since `cesium` is the one who fetches the files under `/Workers` and by that `cesium` lib requirement is to provide statically served files from directly from the `node_modules`

This can be seen on every cesium installation, especially with `webpack` 

https://cesium.com/docs/tutorials/cesium-and-webpack/

See the lines here :

```js
new CopywebpackPlugin([ { from: path.join(cesiumSource, cesiumWorkers), to: 'Workers' } ]),
```

So normally you are correct, the app installing the library should take care of the hashing and caching of the files, but *some file* in cesium are required to be fetched by cesium, so cesium library expects the files to be exist.

Also most of the files under `/Workers` do have cache busting, `cesiumWorkerBootstrapper` is one of the few files which does not.

**Comment by jony89** [comment]
@mramato are you aware of this? I've looked into the gulp file and I saw that `cesiumWorkerBootstrapper` file is excluded from being minified and contain hashed file name, the file is also being hard coded consumed, which is problematic if the filename will contain hash.

**Comment by TJKoury** [comment]
Our distro, [C137.js](https://www.npmjs.com/package/c137.js) can help with this issue since you can cache bust the single file.

**Comment by TJKoury** [comment]
Still working on what could be a solution [here](https://github.com/CesiumGS/cesium/issues/10619).
